modules/log6x-quickstart-viaq.adoc

```diff
@@ -67,22 +69,9 @@ The `ClusterRole` resource is created automatically during the Cluster Logging O
 +
```

Comment:
`Allow the collector's service account to collect logs:`  --> can be updated to something like -->  `To collect logs, use the service account of the collector by running the following command:`

I know this PR doesn't touch this line, but FYI:

Avoid using "allow": https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a

> Avoid stating that inanimate objects grant abilities to people, as in “the product allows you to…”. Whenever possible, use a direct, user-focused alternative such as “you can use the product to…” or “with this product, users can…”.

https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives

> Do not use possessive ’s with inanimate objects.